### DISCUSSION TITLE
Add ability update/delete by query

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#3119] [Feature Requests]
Instead of updating/delete by a known primary key, this allows you to batch update/delete a set of records based on any other given filters.

For example, this would give you the ability to "purge" records effectively by deleting everything from before a given date.

**Comment by wc-matteo** [comment]
or, for example, to update a field inside a translation collection only knowing the main collection pk, filtering by languages_code

Hasura does it like this:

https://hasura.io/docs/latest/graphql/core/api-reference/graphql-api/mutation.html#update-syntax
https://hasura.io/docs/latest/graphql/core/api-reference/graphql-api/mutation.html#delete-syntax

**Comment by br41nslug** [comment]
this was implemented at some point
